releaseArtMaker.sh

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+
```

Comment:
Before starting the script, it is a good idea to check that this is a release branch to avoid inconveniences later. The following should do the trick and exit early:


```
# Check if branch starts with "release"
BRANCH_NAME=$(git rev-parse --abbrev-ref HEAD)
if [[ ! $BRANCH_NAME =~ ^release ]]; then
    echo "Error: This is not a release branch. A release branch has to start with the word \"release\" Exiting."
    exit 1
fi

# Store branch name
RELEASE_BRANCH_NAME=$BRANCH_NAME
```